chainer/testing/parameterized.py

```diff
@@ -92,7 +92,7 @@ def parameterize(*params):
         lambda base: _parameterize_test_case_generator(base, params))
 
 
-def parameterize_pytest(names, values):
+def pytest_to_chainer(names, values):
```

Comment:
I think "chainer" can be inferred from the submodule path. How about `{convert,from}_pytest_params`?

Reply:
That sounds nice.
How about `from_pytest_parameterize`?

Reply:
I suggested "params" as it's the name of the argument to the [`parameterize` function](https://github.com/chainer/chainer/pull/6510/files#diff-f6d4d4f1a30001cf89aca9e815218052R89) but I think either is fine.

Reply:
Fixed.

Reply:
Hm, seems like it's not fixed?

Reply:
Oops, pushed.